### DISCUSSION TITLE
road name overlap from vector map and .sqlitedb offline map

### DISCUSSION THREAD

**Discussion by wu520609** [osmandapp/OsmAnd#18675] [Q&A]
Wu520609

Dec 5, 2023, 13:08 GMT+3

Hello：

I have put my own offline map (.sqlitedb) to osmand.
Yes,It can show my offline map(.sqlitedb). But the road name from vector map also appear. There is 2 road name in one map.So it cause some interference,
Please see attached file(offline.jpg). I have put some red circle for some road name from vector map in offline.jpg.Please take a look. Is any setting we can hide road name from vector map when we use our own offline map(.sqlitedb)?
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/49671574/2a992ff0-cff5-4157-8f57-7e7fca24bd19"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
Do you use raster map as main map source or overlay? You need to use as main map source and in that case text should not be visible to be honest.

**Reply by wu520609** [reply]
Hello:
I use raster map as main map. The text don't appear at first time. But latter(maybe 3 mins),It will appear.

**Reply by vshcherb** [reply]
It's bug obviously, you can also try workaround to use vector layer as main and overlay as raster and set it to non-transparent then symbols from vector map should become transparent

**Comment by wu520609** [comment]
Just provide another info to you.
If I use [Version 1 (OpenGL)],The text don't appear. If I use [Version 2 (OpenGL),The text will appear latter.

**Comment by pfbreton-perso** [comment]
I have this problem as well.

I found that by switching to OpenGL 1 and going back to OpenGL 2 solves it, until the app is restarted.  Then I need to redo this again.

**Reply by vshcherb** [reply]
You can set transperancy to 0 and text will be invisible

**Comment by Raiedecharles** [comment]
same problem
